Ellielee308/wits-travel
# Feature/dashboard userdata

[Website URL](https://witz-ec201--jane-2dqomh6w.web.app/backstage)

## Description
- Added a user usage data chart.
- Changed the static images on the form page to dynamically fetch attraction images.

## Test Plan
- Data Chart: 
    - Access the backend homepage to view the data chart. 
    -  Interact with the frontend webpage and observe changes in the chart data. 
    - Click to download the chart, ensuring that the chart is saved as a PNG file.
- Verify that the contact form's image carousel functionality operates correctly.
